assets/js/blocks/classic-template/index.tsx

```diff
@@ -79,6 +82,12 @@ const Edit = ( {
 							'Removing this will cause unintended effects on your store.',
 							'woo-gutenberg-products-block'
 						) }
+						<br />
+						{ isBlockificationPossible &&
+							__(
+								'We encourage you to use the blockified version of this block instead. Use the button below to migrate.',
```

Comment:
I think that's fine for now, but IMO we should ideally rewrite the whole text displayed in the Classic Template block now. Ie: if we allow people to replace the Classic Template block with the Product, we shouldn't have an alert saying "Do not remove this block!", it feels confusing to me.

![imatge](https://user-images.githubusercontent.com/3616980/214031823-689c4c16-6dcf-4f6a-b8c7-10d4f60cbfbd.png)